src/lsp.rs

```diff
@@ -81,6 +81,99 @@ pub fn get_word_from_pos_params<'a>(
     &line_contents[word_start..word_end]
 }
 
+/// Returns a string slice to the filename in doc specified by the position params
+/// if it is one
```

Comment:
Could you give an example / sample input->result? I'm not sure I understand what this function should return.